src/test/java/seedu/event/commons/util/StringUtilTest.java

```diff
@@ -77,29 +77,29 @@ public void containsWordIgnoreCase_nullSentence_throwsNullPointerException() {
     }
 
     /*
-     * Valid equivalence partitions for word:
-     *   - any word
-     *   - word containing symbols/numbers
-     *   - word with leading/trailing spaces
-     *
-     * Valid equivalence partitions for sentence:
-     *   - empty string
-     *   - one word
-     *   - multiple words
-     *   - sentence with extra spaces
-     *
-     * Possible scenarios returning true:
-     *   - matches first word in sentence
-     *   - last word in sentence
-     *   - middle word in sentence
-     *   - matches multiple words
-     *
-     * Possible scenarios returning false:
-     *   - query word matches part of a sentence word
-     *   - sentence word matches part of the query word
-     *
-     * The test method below tries to verify all above with a reasonably low number of test cases.
-     */
+    * Valid equivalence partitions for word:
+    *   - any word
+    *   - word containing symbols/numbers
+    *   - word with leading/trailing spaces
+    *
+    * Valid equivalence partitions for sentence:
+    *   - empty string
+    *   - one word
+    *   - multiple words
+    *   - sentence with extra spaces
+    *
+    * Possible scenarios returning true:
+    *   - matches first word in sentence
+    *   - last word in sentence
+    *   - middle word in sentence
+    *   - matches multiple words
+    *
+    * Possible scenarios returning false:
+    *   - query word matches part of a sentence word
```

Comment:
Should this case return `true` instead?

Reply:
This test is for `containsWordIgnoreCase` which specifies that the query word must be an exact match of the sentence word. Being part of the sentence word would result in the method returning `false`.
Perhaps you are thinking about the `containsKeywordIgnoreCase` test method? In which case yes it will return true.

Reply:
Oops, sorry, didn't see that.